vkrunner/vr-vk.c

```diff
@@ -71,6 +71,17 @@ vr_vk_load_libvulkan(const struct vr_config *config,
 
         vkfn->vkGetInstanceProcAddr = (void *)
                 GetProcAddress(vkfn->lib_vulkan, "vkGetInstanceProcAddr");
+#elif ANDROID
```

Comment:
Building with GCC 7.3 I get

    vr-vk.c:74:7: warning: "ANDROID" is not defined, evaluates to 0 [-Wundef]